drivers/gpio/gpio_shell.c

```diff
@@ -21,86 +21,75 @@
 
 LOG_MODULE_REGISTER(gpio_shell);
 
-struct args_index {
-	uint8_t port;
-	uint8_t index;
-	uint8_t mode;
-	uint8_t value;
-};
-
-static const struct args_index args_indx = {
-	.port = 1,
-	.index = 2,
-	.mode = 3,
-	.value = 3,
-};
-
 static int cmd_gpio_conf(const struct shell *sh, size_t argc, char **argv)
 {
 	uint8_t index = 0U;
 	int type = GPIO_OUTPUT;
```

Comment:
No need for initial value;
Can be moved closer to usage;
gpio_pin_configure() expects gpio_flags_t type (typedef for uint32_t), not int